open_api_framework/conf/base.py

```diff
@@ -998,12 +997,16 @@ def init_sentry(before_send: Callable | None = None):
     + CORS_ALLOWED_ORIGINS
 )
 
-CSP_IMG_SRC = CSP_DEFAULT_SRC + config(
-    "CSP_EXTRA_IMG_SRC",
-    default=[],
-    split=True,
-    group="Content Security Policy",
-    help_text="Extra ``img-src`` sources for CSP other than ``CSP_DEFAULT_SRC``.",
+CSP_IMG_SRC = (
+    CSP_DEFAULT_SRC
+    + ["data:"]  # used by DRF spectacular
```

Comment:
```suggestion
    + ["data:", "cdn.redoc.ly"]  # used by DRF spectacular
```

It doesn't work for me without this setting